events/events_internal.h

```diff
@@ -32,8 +32,8 @@ struct eventrec * events_immediate_get(void);
  * events_network_select(tv, interrupt_requested):
  * Check for socket readiness events, waiting up to ${tv} time if there are
  * no sockets immediately ready, or indefinitely if ${tv} is NULL.  The value
- * stored in ${tv} may be modified.  If ${interrupt_requested} is non-zero
- * and a signal is sent, exit.
+ * stored in ${tv} may be modified.  If ${*interrupt_requested} is non-zero
```

Comment:
not certain about this -- should we "dereference" the pointer variable in this comment?  I mean, technically any non-NULL `volatile sig_atomic_t *` will be non-zero (unless we're on a really weird platform).

Reply:
Answering my own question: yes, `${*arg}` is the style used on the tarsnap client.

Reply:
Right, the contents of `${}` should be expression-like, so if we want to dereference something we should say that.